### PR TITLE
Update StartNetworkedParticleFxNonLoopedAtCoord.md

### PR DESCRIPTION
**Pull request by VenomXNL**
Provided additional information on how to (properly) call this function, added a small 'example snippet' and information about the visibility for other players when using it networked.

Before you submit this PR, please make sure:

- You have read the contribution guidelines
- You include an example that validates your change
- Your English is grammatically correct

### CODE REVIEW

File: GRAPHICS/StartNetworkedParticleFxNonLoopedAtCoord.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ BOOL START_NETWORKED_PARTICLE_FX_NON_LOOPED_AT_COORD(char* effectName, float xPo
 ```
 
 ```
```
Comment (stannum-cfx) [inline GRAPHICS/StartNetworkedParticleFxNonLoopedAtCoord.md:12]: Please remove the code block from the description.

File: GRAPHICS/StartNetworkedParticleFxNonLoopedAtCoord.md [hunk 2]
```diff
@@ -31,3 +31,33 @@ NativeDB Added Parameter 12: BOOL p11
 * **zAxis**: 
 
 ## Return value
+
+```
```
Comment (stannum-cfx) [inline GRAPHICS/StartNetworkedParticleFxNonLoopedAtCoord.md:35]: Please don't use code blocks in the return value section, if the return value takes more than one line to explain, it's best to put it in the native description instead.

File: GRAPHICS/StartNetworkedParticleFxNonLoopedAtCoord.md [hunk 3]
```diff
@@ -31,3 +31,33 @@ NativeDB Added Parameter 12: BOOL p11
...
+local CurrentPlayerCoords = GetEntityCoords(GetPlayerPed(-1))
+
+UseParticleFxAssetNextCall("scr_indep_fireworks") -- Prepare the Particle FX for the next upcomming Particle FX call
+SetParticleFxNonLoopedColour(1.0, 0.0, 0.0) -- Setting the color to Red (R, G, B)
+StartNetworkedParticleFxNonLoopedAtCoord("scr_indep_firework_burst_spawn", CurrentPlayerCoords, 0.0, 0.0, 0.0, 1.0, false, false, false, false) -- Start the animation itself
+
+RemoveNamedPtfxAsset("scr_indep_fireworks") -- Clean up
+```
+
+## Large Particle List:
```
Comment (stannum-cfx) [inline GRAPHICS/StartNetworkedParticleFxNonLoopedAtCoord.md:58]: Please don't add new sections with (`##`) headers , place any information you have in the native description instead.

**Review by stannum-cfx — CHANGES_REQUESTED** [review]

**Comment by VenomXNL** [comment]
No longer working (or planning to work) on FiveM or related mod's, and IF i will do so it from here on will only be my own externally released "Works".

**Pull request closed without merging**